src/main/webapp/courseLoader.js

```diff
@@ -1,127 +1,200 @@
+let termData;
 google.charts.load("current", { packages: ["corechart"] });
-google.charts.setOnLoadCallback(makeGraphs);
 
 function fillTitles() {
-    const queryString = window.location.search;
-    const urlParams = new URLSearchParams(queryString);
-    const courseName = urlParams.get("course-name");
-    const term = urlParams.get("term");
-    document.getElementById("course-name").innerHTML = courseName;
-    document.getElementById("term-name").innerHTML = term;
+  const queryString = window.location.search;
+  const urlParams = new URLSearchParams(queryString);
+  const courseName = urlParams.get("course-name");
+  const term = urlParams.get("term");
+  document.getElementById("course-name").innerHTML = courseName;
+  document.getElementById("term-name").innerHTML = term;
 }
 
-function populateData(){
-    const queryString = window.location.search;
-    const urlParams = new URLSearchParams(queryString);
-    const courseName = urlParams.get("course-name");
-    const term = urlParams.get("term");
-    const profName = urlParams.get("prof-name");
-    const units = urlParams.get("num-units")
-    const schoolName = urlParams.get("school-name")
+function populateData() {
+  const queryString = window.location.search;
+  const urlParams = new URLSearchParams(queryString);
+  const courseName = urlParams.get("course-name");
+  const term = urlParams.get("term");
+  const profName = urlParams.get("prof-name");
+  const units = urlParams.get("num-units");
+  const schoolName = urlParams.get("school-name");
 
-  fetch(`/term-live?school-name=${schoolName}&course-name=${courseName}&term=${term}&prof-name=${profName}&num-units=${units}`)
+  fetch(
+    `/term-live?school-name=${schoolName}&course-name=${courseName}&term=${term}&prof-name=${profName}&num-units=${units}`
+  )
     .then((response) => response.json())
-    .then((data => {
-        makeGraphs(data);
-    }));
+    .then((data) => {
+      termData = data;
```

Comment:
This is the only usage of termData I see, why not just pass it through directly? Was this a test only variable?

Reply:
I left it like this for the future because I envisioned a case where I would need it elsewhere, but for the time being I will remove it until I need it.